recipe/build.sh

```diff
@@ -7,13 +7,12 @@ then
 fi
 
 cmake -LAH -G "Ninja" \
-  ${CMAKE_ARGS} \
   -DCMAKE_PREFIX_PATH=${PREFIX} \
   -DCMAKE_FIND_FRAMEWORK=LAST \
   -DCMAKE_INSTALL_RPATH:STRING=${PREFIX}/lib \
   -DCMAKE_UNITY_BUILD=ON \
   -DCMAKE_UNITY_BUILD_BATCH_SIZE=32 \
   -DCMAKE_MESSAGE_LOG_LEVEL=STATUS \
   -DQT_HOST_PATH=${PREFIX} \
-  -B build  .
+  -B build ${CMAKE_ARGS} .
```

Comment:
wow this order matter.s...

Reply:
yes QT_HOST_PATH is passed twice: once at the line below and once via CMAKE_ARGS when cross-compiling so the order matter for the cross-compiling one to override the first